docs/usage/types/booleans.md

```diff
@@ -5,23 +5,17 @@ description: Support for Boolean types.
 `bool`
 : see below for details on how bools are validated and what values are permitted
 
-!!! warning
```

Comment:
This warning was for v1.0; it seems to me safe to remove this now, especially since older docs versions will still be accessible.

Reply:
I'd agree. @tpdorsey ?

Reply:
Agree